ppl/cmd/zqd/listen/command.go

```diff
@@ -83,6 +94,10 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	f.StringVar(&c.suricataUpdaterPath, "suricataupdater", "", "command to update Suricata rules (run once at startup)")
 	f.StringVar(&c.zeekRunnerPath, "zeekrunner", "", "command to generate Zeek logs from pcap data")
 
+	// Hidden flag while we transition to using archive store by default.
+	// See zq#1085
+	f.Var(cli.FuncValue(setApiFileStoreReadOnly), "filestorereadonly", "boolean to make file store spaces read only (and use archive store by default)")
```

Comment:
s/Api/API/

But this would all be simpler as
```go
f.BoolVar(&api.FileStoreReadOnly , "filestorereadonly", false, "make file store spaces read only (and default to archive store)")
```